tests/lib/tokenize.js

```diff
@@ -199,4 +199,16 @@ describe("tokenize()", () => {
         espree.tokenize("foo", Object.freeze({ ecmaFeatures: Object.freeze({}) }));
     });
 
+    it("should produce tokens when } is the last token", () => {
```

Comment:
Do you mind adding a comment with a link to the relevant issue here so that future contributors have some context as to why this test exists?

Reply:
I've added, but I'm not sure about the description

Reply:
Sorry, could have been clearer. This is fine, but I also think a comment that just says `// https://github.com/eslint/espree/issues/403` would suffice!

Reply:
Sorry, if you would suggest me what to add, I will add

Reply:
This is fine! Thanks for adding the comment.